projects/kit/components/input-year/input-year.component.ts

```diff
@@ -88,8 +84,20 @@ export class TuiInputYearComponent
         return this.options.icon;
     }
 
+    @tuiPure
+    getMaskOptions(max: number): MaskitoOptions {
+        const numberOptions = maskitoNumberOptionsGenerator({
+            max,
+            precision: 0,
+            isNegativeAllowed: false,
```

Comment:
Do `precision` and `isNegativeAllowed` matter if we override the `mask` anyway?

Reply:
You are right, I should drop it.

Reply:
fixed